bsvalias/site.go

```diff
@@ -13,27 +13,48 @@ func GetSite(ctx context.Context, domain string) (Site, error) {
 	_, records, _ := net.LookupSRV("bsvalias", "tcp", domain)
 
 	var site Site
+
 	if len(records) > 0 {
-		// Strip period at end of target if it exists.
-		// I am not sure why it is there --ce
-		l := len(records[0].Target)
-		if records[0].Target[l-1] == '.' {
-			records[0].Target = records[0].Target[:l-1]
+		// Strip period at end of target.
+		r := records[0]
```

Comment:
Do we really need all these blank lines?

Reply:
Fixed.